src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -349,7 +370,8 @@ private static IEnumerable GetAsmInstallations(string rootFolder)
                 throw new MissingMethodException("Method 'DynamoInstallDetective.Utilities.FindProductInstallations' not found");
             }
 
-            var methodParams = new object[] { "Revit", "ASMAHL*.dll" };
+
```

Comment:
Could you use ASMFileMask from above to avoid string duplication?

Reply:
Will do a search and replace